tests/core/web3-module/test_sha3.py

```diff
@@ -11,7 +11,8 @@
     [
         ('cowmö', HexBytes('0x0f355f04c0a06eebac1d219b34c598f85a1169badee164be8a30345944885fe8')),
         ('', HexBytes('0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470')),
-    ]
+    ],
+    ids=['test_' + str(x) for x in range(2)]
```

Comment:
These `ids` can go too (although the trailing comma on the list of args can stay).